### PR TITLE
#166790015 Users should be able to see their reading stats

### PR DESCRIPTION
**Pull request by Hervera**
#### What does this PR do?
Enables users to see their reading stats

#### Description of Task to be completed
- Record reading
- Get user reading stats

#### How should this be manually tested?
1. Run the following commands in the terminal
```
git clone https://github.com/andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ft-users-reading-stats-166790015
npm install
npm start
```
2. Create a .env file and add the following snippets
```
DATABASE_URL=postgres://username:password@localhost:5432/authorshaven
DATABASE_=postgres://username:password@localhost:5432/testing-ah
SECRET=whatever you want it to be
```
3. Install `sequelize` globally
```
sudo npm install -g sequelize
```
4.  Create a database using `sequelize db:create`
5. Install and setup Postman to consume CRUD endpoints
6. Run the following requests in Postman using `localhost:3000`
## Endpoints
|  Method  |  Endpoint  |  Task  |
|  --- |  --- |  ---  |
|  `POST`  |  `/api/articles/:articleSlug/record-reading`  |  `record reading`  |
|  `GET`  |  `/api/users/reading-stats`  |  `reading stats`  |

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166790015](https://www.pivotaltracker.com/story/show/166790015)

#### Screenshots (if appropriate)
`Record reading`
Screen Shot 2019-07-26 at 13 30 22

`Reading stats`
Screen Shot 2019-07-26 at 13 30 36

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Review by niyoceles — DISMISSED** [review]

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by IsaiahRn — DISMISSED** [review]
It can be merged

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by niyoceles — DISMISSED** [review]

**Review by niyoceles — DISMISSED** [review]

**Review by IsaiahRn — DISMISSED** [review]

**Review by elemanhillary-zz — DISMISSED** [review]
ready for review

**Review by niyoceles — DISMISSED** [review]

**Review by IsaiahRn — DISMISSED** [review]

**Review by elemanhillary-zz — DISMISSED** [review]
ready for merge

**Review by IsaiahRn — APPROVED** [review]

**Review by IsaiahRn — DISMISSED** [review]

**Review by niyoceles — DISMISSED** [review]

**Review by kodek-sleuth — DISMISSED** [review]

File: controllers/readingStat.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          ['updatedAt', 'lastTime']],
+        include: [
+          {
+            model: articles,
+            attributes: ['title', 'slug', 'body'],
+          }
+        ]
+      });
+      if (totalUserReadingStats) {
+        return res.status(200).send({ totalArticlesRead: totalUserReadingStats, articlesRead });
```
Comment (Hervera) [inline controllers/readingStat.js]: Use `json()` instead of `send()`

**Review by IsaiahRn — APPROVED** [review]
ready to be merged

**Review by niyoceles — APPROVED** [review]

**Review by kodek-sleuth — APPROVED** [review]

**Pull request merged**